lexical-write-integer/src/jeaiii.rs

```diff
@@ -225,8 +271,51 @@ pub fn from_u32(n: u32, buffer: &mut [u8]) -> usize {
     }
 }
 
+/// Optimized jeaiii algorithm for u64.
+#[inline(always)]
+#[allow(clippy::collapsible_else_if)] // reason = "branching is fine-tuned for performance"
+pub fn from_u64(n: u64, buffer: &mut [u8]) -> usize {
+    // NOTE: Like before, this optimizes better for large and small
+    // values if there's a flat comparison with larger values first.
+    const FACTOR: u64 = 100_0000_0000;
+    let buffer = &mut buffer[..20];
```

Comment:
@Alexhuszagh Hello, I got a panic in the recent release `lexical-write-integer-1.0.4`. I guess the error is due to the change of buffer size

Reply:
That's my bad, I'll fix this right now. `i64` only requires 19 digits, `u64` requires 20.